includes/class-amp-story-post-type.php

```diff
@@ -646,4 +686,29 @@ public static function the_single_story_card( $post ) {
 		</a>
 		<?php
 	}
+
+	/**
+	 * Enqueues this post type's stylesheet for the embed endpoint.
+	 */
+	public static function enqueue_embed_styling() {
+		if ( is_embed() && is_singular( self::POST_TYPE_SLUG ) ) {
+			wp_enqueue_style(
```

Comment:
This style is being registered in two places. Can it be registered in one and then enqueued in two?

Reply:
Good idea.

Reply:
https://github.com/ampproject/amp-wp/pull/1944/commits/a9b5fdbdfa72ef3d0d2b81bfc6645f61b9be57c1 registers the CSS file on the `wp_loaded` hook. Embed endpoints don't seem to fire `wp_enqueue_scripts`. 

It's not ideal, but `wp_loaded` was the best hook I could find that works for the embed and the Latest Stories block.

Reply:
@kienstra you can use the `wp_default_scripts` action instead, perhaps including the CSS registration in this function:

https://github.com/ampproject/amp-wp/blob/44a94ce43ac95ca853cc6502a113286cb7901a8b/includes/amp-helper-functions.php#L377-L450